Sources/VLCEditController.swift

```diff
@@ -235,3 +253,23 @@ extension VLCEditController: UICollectionViewDelegateFlowLayout {
         return CGSize(width: collectionView.frame.width - insetToRemove, height: MediaEditCell.height)
     }
 }
+
+extension VLCEditController: AddToPlaylistViewControllerDelegate {
+    func addToPlaylistViewController(_ addToPlaylistViewController: AddToPlaylistViewController,
+                                     didSelectPlaylist playlist: VLCMLPlaylist) {
+        let files = model.anyfiles
+        for index in selectedCellIndexPaths  where index.row < files.count {
```

Comment:
double space I'm surprised swift lint didn't catch that

Reply:
Indeed!